x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/connector_configuration_form.tsx

```diff
@@ -48,6 +49,12 @@ export const ConnectorConfigurationForm = () => {
     {}
   );
 
+  const filteredConfigView = localConfigView.filter(
```

Comment:
nitpicking after the fact, but this should be done in a selector in the logic file where `localConfigView` originated to take advantage of caching (otherwise this recomputes on every unrelated re-render).

Reply:
I'll fix this in a followup PR.

Reply:
Followup: https://github.com/elastic/kibana/pull/155608